topics/epigenetics/tutorials/tal1-binding-site-identification/tutorial.md

```diff
@@ -6,22 +6,22 @@ redirect_from:
 title: "Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1)"
 zenodo_link: "https://doi.org/10.5281/zenodo.197100"
 edam_ontology: "topic_3169"
-enable: false
+enable: true
```

Comment:
```suggestion
```

is seems there's a small bug in our linting for whenever this value is 'true', so, we can just remove this attribute and it will have the same effect.

Reply:
I was trying to read through the Travis logs and couldn't figure out what the bug was. Thanks for finding it!